ui/src/app/pages/workspace/workspace-wrapper/component.ts

```diff
@@ -94,8 +98,11 @@ export class WorkspaceWrapperComponent implements OnInit, OnDestroy {
             this.setSidebarState(false);
           }
         }));
-    this.subscriptions.push(routeConfigDataStore.subscribe(({minimizeChrome}) => {
-      this.displayNavBar = !minimizeChrome;
+    this.subscriptions.push(routeConfigDataStore.subscribe((data) => {
+      // ROUTER MIGRATION: Prevent dueling route configs during react transition - only apply minimizeChrome if there is route data
```

Comment:
After the full React conversion, would this check be removed, or no? May be worth specifying.

Reply:
I removed the "ROUTER CONVERSION" portion. After thinking it through some more, this entire component will no longer exist. It will either need to be ported over to react or removed in favor of another strategy.